components/expenses/Expense.js

```diff
@@ -500,8 +505,50 @@ class Expense extends React.Component {
           {this.state.error && (
             <MessageBox type="error" withIcon mb={2}>
               {formatErrorMessage(intl, this.state.error)}
+              <br />
+              {formatErrorMessage(intl, this.state.error).startsWith(
+                'Host has two-factor authentication enabled for large payouts.',
```

Comment:
`formatErrorMessage` is intended to translate the error message, so we shouldn't compare its result to check the error type.